devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -187,7 +210,6 @@ private void registerTasks(Project project, QuarkusPluginExtension quarkusExt) {
 
         });
 
-        configureBuildNativeTask(project);
```

Comment:
What happens now if BUILD_NATIVE_TASK_NAME is executed?

Reply:
Hm - wonder whether it's okay to remove both `buildNative` + `testNative`, as those are already deprecated?

Reply:
While it's deprecated, I suspect people will ask for it if we remove it. If I'm not mistaken it happened before. Could we keep it for now?

Reply:
Re-added it and adopted the native-integration-tests a little (not sure why those didn't fail).